src/Element/BaseInput.php

```
@@ -111,9 +110,18 @@ abstract protected function getTag(): VoidInterface;
      */
     protected function buildElement(string|Stringable $content = '', array $tokenValues = []): string
     {
+        $attributes = $this->getAttributes();
+
+        /** @phpstan-var string|null $id */
+        $id = $this->getAttribute('id', null);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)